src/commands/command.ts

```diff
@@ -0,0 +1,19 @@
+import {ArgDescriptor} from 'command-line-args';
+
+export interface Command {
+  name: string;
+  description: string;
+  args: ArgDescriptor[];
+  run(options: {[name: string]: string}): Promise<any>;
+}
+
+// TODO(justinfagnani): how do I re-export an interface?
+export interface ArgDescriptor extends ArgDescriptor {
```

Comment:
hmm... it's not anymore. I was pulling it in with `Command` into polytool and help. I might again.